Nos/Models/FlagOption.swift

```diff
@@ -17,46 +17,42 @@ struct FlagOption: Identifiable, Equatable {
-        FlagOption(
-            title: String(localized: .localizable.flagContentSpamTitle),
-            description: nil,
-            info: nil,
-            category: .report(ReportCategoryType.spam)
-        ),
-        FlagOption(
-            title: String(localized: .localizable.flagContentHarassmentTitle),
-            description: String(localized: .localizable.flagContentHarassmentDescription),
-            info: nil,
-            category: .report(ReportCategoryType.harassment)
-        ),
-        FlagOption(
-            title: "NSFW",
-            description: String(localized: .localizable.flagContentNudityDescription),
-            info: nil,
-            category: .report(ReportCategoryType.nsfw)
-        ),
+    /// Creates a list of `FlagOption` categories based on the provided flag target.
+    /// - Parameter flagTarget: The target of the report.
+    /// - Returns: An array of `FlagOption` categories,  modified based on the `flagTarget`.
+    static func createFlagCategories(for flagTarget: ReportTarget) -> [FlagOption] {
+        var categories = [spamCategory, harassmentCategory, nsfwCategory, illegalCategory, otherCategory]
+
+        if case .author = flagTarget {
+            let insertionIndex = max(categories.count - 1, 0)
+            categories.insert(impersonationCategory, at: insertionIndex)
+        }
+
+        return categories
+    }
+
+    /// `FlagOption` instances representing different categories of how a content can can be flagged.
+    static let flagContentSendCategories: [FlagOption] = [
         FlagOption(
-            title: String(localized: .localizable.flagContentIllegalTitle),
-            description: String(localized: .localizable.flagContentIllegalDescription),
-            info: nil,
-            category: .report(ReportCategoryType.illegal)
+            title: String(localized: .localizable.flagContentSendToNosTitle),
+            description: String(localized: .localizable.flagContentSendToNosDescription),
+            info: String(localized: .localizable.flagContentSendToNosInfo),
+            category: .privacy(.sendToNos)
         ),
         FlagOption(
-            title: String(localized: .localizable.flagContentOtherTitle),
-            description: String(localized: .localizable.flagContentOtherDescription),
+            title: String(localized: .localizable.flagContentFlagPubiclyTitle),
+            description: String(localized: .localizable.flagContentFlagPubiclyDescription),
             info: nil,
-            category: .report(ReportCategoryType.other)
+            category: .privacy(.publicly)
         )
     ]
 
-    /// `FlagOption` instances representing different categories of how a content can can be flagged.
-    static let flagContentSendCategories: [FlagOption] = [
+    /// `FlagOption` instances representing different categories of how an account can be flagged.
+    static let flagAccountSendCategories: [FlagOption] = [
```

Comment:
I might call this "flagAccountSendOptions"
```suggestion
    static let flagAccountSendOptions: [FlagOption] = [
```